docs/volumes.md

```diff
@@ -1,100 +1,182 @@
 # Volumes
-This document describes the current state of Volumes in kubernetes.  Familiarity with [pods](./pods.md) is suggested.
 
-A Volume is a directory, possibly with some data in it, which is accessible to a Container. Kubernetes Volumes are similar to but not the same as [Docker Volumes](https://docs.docker.com/userguide/dockervolumes/).
-
-A Pod specifies which Volumes its containers need in its [spec.volumes](http://kubernetes.io/third_party/swagger-ui/#!/v1/createPod) property.
-
-A process in a Container sees a filesystem view composed from two sources: a single Docker image and zero or more Volumes.  A [Docker image](https://docs.docker.com/userguide/dockerimages/) is at the root of the file hierarchy.  Any Volumes are mounted at points on the Docker image;  Volumes do not mount on other Volumes and do not have hard links to other Volumes.  Each container in the Pod independently specifies where on its image to mount each Volume.  This is specified in each container's VolumeMounts property.
+On-disk files in a container are ephemeral.  If that container crashes and
+restarts, the files are wiped, and the container starts with a clean slate.
+Running containers together in a pod, it is frequently necessary to share files
+between those containers.  The Kubernetes `volume` abstraction solves both of
+these problems.
+
+Familiarity with [pods](./pods.md) is suggested.
+
+## Background
+
+Docker also has a concept of
+[volumes](https://docs.docker.com/userguide/dockervolumes/), though it is
+somewhat looser and less managed.  In Docker, a volume is simply a directory on
+disk or in another container.  Lifetimes are not managed and until very
+recently there were only local-disk-backed volumes.  Docker now provides volume
+drivers, but the functionality is very limited, still.
+
+A Kubernetes volume, on the other hand, has an explicit lifetime - the same as
+the pod that encloses it.  When a Pod ceases to exist, the volume will cease to
```

Comment:
"the pod that encloses it.  Consequently, it outlives an containers that run within the Pod, data is preserved across Container restarts.  Of course, when a Pod ceases to exist..."